nilearn/plotting/surf_plotting.py

```diff
@@ -21,7 +21,7 @@
 from nilearn.plotting._utils import check_surface_plotting_inputs
 from nilearn.plotting.cm import cold_hot, mix_colormaps
 from nilearn.plotting.displays._figures import PlotlySurfaceFigure
-from nilearn.plotting.displays._slicers import _get_cbar_ticks
+from nilearn.plotting.displays._slicers import get_cbar_ticks
```

Comment:
As this is accessed from outside `nilearn.plotting.displays` shouldn't it be declared in `__all__` in `nilearn/plotting/displays/__init__.py`  and `nilearn.plotting.displays._slicers`?

Reply:
> shouldn't it be declared in __all__ in nilearn/plotting/displays/__init__.py

nope because in this case that would be making it part of the user facing API